### PR TITLE
[WIP] Translation community handbook

### PR DESCRIPTION
**Pull request by acocac**
### Summary
Documenting general translation guidelines for the community handbook.

### List of changes proposed in this PR (pull-request)

* *Create a translation landing page with general guidelines.*

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok? 
- [ ] Can we get some content on how to sync a new language in transifex? 

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/_toc.yml [hunk 1]
```diff
@@ -512,6 +512,8 @@
       file: community-handbook/coworking/coworking-weekly
     - title: Organising coworking calls
       file: community-handbook/coworking/coworking-organisation
+  - title: Translation
```
Comment (malvikasharan) [inline book/website/_toc.yml:515]:
```suggestion
  - title: Translating The Turing Way
```

**Comment by malvikasharan** [comment]
@acocac this chapter's structure and content are looking great. I can help draft an overview in this branch, but please do continue editing the current chapter on the process in your own time. I will work on this a bit at the collaboration cafe this week, so do leave me some specific pointers where my contribution/edit will be useful.

**Comment by malvikasharan** [comment]
@all-contributors please add @acocac for content

**Comment by malvikasharan** [comment]
Sharing this resource from Yanina Saibene (@yabellini): https://github.com/gvwilson/teachtogether.tech/blob/main/es/README.md

**Comment by da5nsy** [comment]
Closing as it looks like this was superseded by #3153.
Feel free to reopen if I've misunderstood.

**Pull request closed without merging**